design/ceph/object/ceph-rgw-support-multisite-for-external-cluster.md

```diff
@@ -0,0 +1,50 @@
+---
+# Improvements to RGW Multisite configuration
+target-version: release-1.10
+---
+
+# Feature Name
+ Improvements to RGW Multisite configuration
+
+## Summary
+Following are the major changes planned with this feature:
+* For external RGW servers, have place holder for zone/zonegroup/realm settings in `GatewaySpec`.
+* Provide option to add Endpoints to `zone` configuration.
+* Encode the `Endpoint` information into `realm-system-user` secret using the RGW Kubernetes service name.
+
+### Goals
+Improve usability of RGW mulitisite for customers and make it more friendly for disaster recovery scenarios. And planning to cover following use-cases:
+* two ceph object store clusters running in two different kubernetes cluster.
+* one ceph object store resides in kubernetes and another is external.
```

Comment:
Aren't these two cases essentially equivalent? In both cases, won't there be one CephObjectStore in external mode?